tests/ocpp_tests/run-testing.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+source ./run-testing-header.sh
+
+echo "Running $PARALLEL_TESTS tests in parallel"
+
+# run all tests in parallel
+"$PYTHON_INTERPRETER" -m pytest -d --tx "$PARALLEL_TESTS"*popen//python="$PYTHON_INTERPRETER" -rA --junitxml=result.xml --html=report.html --self-contained-html --max-worker-restart=0 --timeout=300 --dist loadgroup test_sets/ocpp16/*.py test_sets/ocpp201/*.py --everest-prefix "$EVEREST_CORE_DIR/build/dist"
```

Comment:
It is nice to be able to run all tests in parallel. Is it possible to add this as an environment variable so you can choose how many processors are used? Instead of always using all of them for the tests.

Reply:
you can already run e.g.

```bash
./run-testing.sh 4
```

to run with 4 workers.